packages/core/admin/packup.config.ts

```diff
@@ -0,0 +1,52 @@
+import { Config, defineConfig } from '@strapi/pack-up';
+import { transformWithEsbuild } from 'vite';
+
+const config: Config = defineConfig({
+  bundles: [
+    {
+      source: './admin/src/index.ts',
+      import: './dist/admin/index.mjs',
+      require: './dist/admin/index.js',
+      types: './dist/admin/src/index.d.ts',
+      tsconfig: './admin/tsconfig.build.json',
+      runtime: 'web',
+    },
+    {
+      source: './_internal/index.ts',
+      import: './dist/cli.mjs',
+      require: './dist/cli.js',
+      // types: './dist/cli/index.d.ts',
+      // tsconfig: './tsconfig.build.json',
```

Comment:
Are these supposed to be here?

Reply:
I can remove them for now – because of circular dependencies I can't build types for the commands we export :(